src/Lecture3.hs

```diff
@@ -137,10 +149,11 @@ data List1 a = List1 a [a]
 
 -- | This should be list append.
 instance Semigroup (List1 a) where
-
+  (<>) :: List1 a -> List1 a -> List1 a
+  (<>) (List1 x xs) (List1 y ys) = List1 x (xs ++ (y:ys))
 
 {- | Does 'List1' have the 'Monoid' instance? If no then why?
-
+-- No there is no monoid instance because List x [] <> List y ys = List x ys which breaks the law of right identity
```

Comment:
That's correct! 👍🏻 
Or, in other words, there's no such thing as "empty non-empty list" 🙂